src/py42/clients/detectionlists/high_risk_employee.py

```diff
@@ -0,0 +1,265 @@
+import json
+
+from py42.clients import BaseClient
+from py42.clients.util import get_all_pages
+
+
+class HighRiskEmployeeClient(BaseClient):
+    """
+    Administrator utility to manage High Risk employees.
+    `Support Documentation <https://support.code42.com/Administrator/Cloud/Monitoring_and_managing/Detection_list_management_APIs>`__
+
+    """
+
+    # In other modules URI is a single variable, breaking it up to have better control
+    # if we want to refactor version and prefix to a common configuration.
+    _api_version = u"v2"
+    _uri_prefix = u"/svc/api/{0}".format(_api_version)
+
+    def __init__(self, session, user_context):
+        super(HighRiskEmployeeClient, self).__init__(session)
+        self._user_context = user_context
+
+    def _add_high_risk_employee(self, tenant_id, user_id):
+
+        data = {"tenantId": tenant_id, "userId": user_id}
+        resource = u"/highriskemployee/add"
+        uri = u"{0}{1}".format(self._uri_prefix, resource)
+        return self._session.post(uri, data=json.dumps(data))
+
+    def add(self, user_id=None):
+        """
```

Comment:
Once they are added to `DetectionListUserClient`, use the `get` method to see if the id has already been created and `create` method if it has not, _then_ call `add_high_risk_employee()`.

Reply:
cannot call create method, since we need `username` to call it, just added a check to verify user_id exists in detection list.